src/Microsoft.Diagnostics.Runtime/src/DataReaders/DbgEng/DbgEngDataReader.cs

```diff
@@ -219,7 +220,7 @@ public IEnumerable<ModuleInfo> EnumerateModules()
                 for (int i = 0; i < bases.Length; ++i)
                 {
                     string? fn = _symbols.GetModuleNameStringWide(DebugModuleName.Image, i, bases[i]);
-                    ModuleInfo info = new ModuleInfo(bases[i], mods[i].Size, mods[i].TimeDateStamp, fn, isVirtual: true);
+                    ModuleInfo info = new ModuleInfo(bases[i], mods[i].Size, mods[i].TimeDateStamp, fn, isVirtual: true, buildId: ImmutableArray<byte>.Empty);
```

Comment:
Maybe we should just remove the buildId parameter and always lazily calculate it?

Reply:
Why lazily calculate if the build ID is known to be `Empty` on Windows?

Reply:
Ah right.  =)